src/mathtransfo.cpp

```diff
@@ -0,0 +1,53 @@
+﻿#include "mathtransfo.h"
+
+float radToDeg(float x)
+{
+	return x*180/PI;
+}
+
+float degToRad(float x)
+{
+	return x/180*PI;
+}
+
+float normalizeDeg(float x)
+{
+	return x/359;
```

Comment:
Why isn't it 360?
One entire revolution is 360.0 degres, no ?
`x` is not an integer, it;s a floating point. So I _think_ this should be `360` here.